host-controller/src/main/java/org/jboss/as/host/controller/Main.java

```diff
@@ -564,24 +564,18 @@ private static URL makeURL(String urlspec) throws MalformedURLException {
             for (String arg : runtime.getInputArguments()) {
                 if (arg != null && arg.length() > 2 && arg.startsWith("-D")) {
                     arg = arg.substring(2);
-                    String[] split = arg.split("=");
-                    if (!hostSystemProperties.containsKey(split[0])) {
```

Comment:
With this refactoring the semantics of this method didn't change (it is backward compatible).
What this code does it it fixes the handling of special -XaddExport parameters,
which have the form jdk9.module.name1=package1,jdk9.module.name2=package2.export.